mk/mcu/AT32F4.mk

```diff
@@ -50,7 +49,7 @@ DEVICE_FLAGS   += -DUSE_ATBSP_DRIVER -DAT32F43x -DHSE_VALUE=$(HSE_VALUE) -DAT32
 
 MCU_COMMON_SRC = \
     $(addprefix startup/at32/,$(notdir $(wildcard $(SRC_DIR)/startup/at32/*.c))) \
-    $(addprefix drivers/at32/,$(notdir $(wildcard $(SRC_DIR)/drivers/at32/*.c))) \
+    $(addprefix drivers/mcu/at32/,$(notdir $(wildcard $(SRC_DIR)/drivers/mcu/at32/*.c))) \
```

Comment:
Another PR may replace this with list of files

Reply:
Done.